tools/cli/util.go

```diff
@@ -667,6 +667,7 @@ func processJSONInputHelper(c *cli.Context, jType jsonType) string {
 		input = c.String(flagNameOfRawInput)
 	} else if c.IsSet(flagNameOfInputFileName) {
 		inputFile := c.String(flagNameOfInputFileName)
+		// #nosec
```

Comment:
@vancexu please review and provide comment.

Reply:
The added comment is correct